### PR TITLE
Feature/connect project view to backend

### PR DESCRIPTION
**Pull request by KristinaGru**
## 📝 Description

- Switch from using mock data in project task view page to real database
- Added get, post and put request for tasks and get request for projects

## ⚠️ Notes

- Using local DB
- Making a PR to 'get-db-data' branch, because it has the backend setup

## 🎉 Demos

image

### CODE REVIEW

**Review by kralle333 — CHANGES_REQUESTED** [review]
I would incur you to wait for the backend feature to get merged in to develop before you make this PR. When it has been merged in, you can make a new feature branch with develop as base and cherry pick these commits or squash merge this branch.

**Comment by KristinaGru** [comment]
> I would incur you to wait for the backend feature to get merged in to develop before you make this PR. When it has been merged in, you can make a new feature branch with develop as base and cherry pick these commits or squash merge this branch.

@kralle333 Noted! Just for this time, since the 'get-db-data' branch got merged, would it be fine to leave this PR as is, since it switched to dev as base? This branch is up to date with the current dev, and all the commits here are what I would want to merge to it

File: frontend/src/pages/projectView/ProjectView.tsx [hunk 1]
```diff
@@ -39,35 +62,38 @@ const ProjectView = () => {
...
+        title,
+        description: '',
+        status,
+        due_date: null,
+        user_uid: userId,
+        assignee: '',
+        completed: false,
+        priority: 'medium',
+        project_id: +id,
+      };
```
Comment (kralle333) [inline frontend/src/pages/projectView/ProjectView.tsx]: maybe you wanna make a constructor for this? Also rename id into project_id

**Review by kralle333 — APPROVED** [review]
Really nice work, I liked that you renamed user_id to user_uid.

I have one small comment, but it is not blocking

**Pull request merged**